modules/olm-installing-operators-from-operatorhub.adoc

```diff
@@ -1,6 +1,7 @@
 // Module included in the following assemblies:
 //
 // * operators/olm-adding-operators-to-cluster.adoc
+// * post_installation_configuration/preparing-for-users.adoc
 
 [id="olm-installing-operators-from-operatorhub_{context}"]
```

Comment:
I think it's important here to mention that when User choose "All Namespaces", the operator is being installed at openshift-operators namespace and not all operators are supported to be installed in that way.